### DISCUSSION TITLE
Update config.guess and config.sub?

### DISCUSSION THREAD

**Discussion by kklobe** [PCemOnMac/PCemV17macOS#18] [Ideas]
**config.guess** and **config.sub** are very old (2014), and I noticed some spot fixes to allow Apple Silicon builds. A more forward-looking solution might be to update those files to current versions. Just doing that allowed me to compile mainline, for example, with no further tweaks.

https://www.gnu.org/software/gettext/manual/html_node/config_002eguess.html

**Comment by kyr0** [comment]
Hi Kirk,

nice, good to know. I think this could be very helpful for our backport to mainline. Do you have a working example maybe? :)

Thank you in advance! :)

Aron

**Reply by kklobe** [reply]
I just updated those two files, after cloning this repo, using the `wget` commands in the link above and ran ./configure && make.
